spec/lib/excel_upload_spec.rb

```diff
@@ -3,28 +3,28 @@
 
 describe ExcelUpload do
   before :each do
-    @settings = Settings.import
+    @first_row = 4
```

Comment:
woops - I should be getting the class variable here instead of assigning @first_row to 4 directly.